content/en/docs/contribute/localization.md

```diff
@@ -187,8 +187,11 @@ script and use it in the theme. Assign "language name in latin script" to
 `languageNameLatinScript`. For example, `languageNameLatinScript ="Korean"` or
 `languageNameLatinScript = "Deutsch"`.
 
-When assigning a `weight` parameter for your block, find the language block with
-the highest weight and add 1 to that value.
+The `weight` parameter determines the order of languages in the language selection bar.
+A lower weight takes precedence, resulting in the language appearing first. 
+When assigning the `weight` parameter, it is important to examine the existing languages 
+block and adjust their weights to ensure they are in a sorted order relative to all languages,
```

Comment:
Why can't we reserve some buffer numbers in between that may be useful in the future for upcoming languages?
This way we can avoid existing code edit.

I hope my question is valid.

Reply:
It's a valid consideration, one that I even pondered while implementing this change. But I feel maintaining a sorted order list _(proper sequenced weight)_ for languages does seem more practical in the long run. Predicting the exact number of future languages can be challenging, and allocating buffer numbers could lead to uneven distribution over time, potentially causing confusion. One more thing I will do _(now)_ is organize language blocks itself in sorted order, which would help in keeping things organized and easier to manage in the long run.

Reply:
Agree with this,
Thanks.